Week1/homework/js-exercises/LengthString.js

```diff
@@ -0,0 +1,4 @@
+"use strict";
+var mySentence = 'Programming is so interesting!';
+var lngth = mySentence.length;
```

Comment:
`const` instead of `var` here please. Also, there is a spelling mistake for variable name `length`. 🙂